### ISSUE TITLE
Unable to turn on/off dimmer devices

### ISSUE THREAD

**Issue by shumonsharif** [FlyingDiver/Indigo-Masquerade#1]
Hi Joe,

Thanks for yet another wonderful plugin, I'm able to add Alexa volume control to all my airplay / Airfoil speakers.  However, I was unable to turn on or off the masqueraded dimmer devices -- with the following two exceptions appearing in my logs:

```
Traceback (most recent call last):
  File "plugin.py", line 286, in actionControlDevice
KeyError: key highLimit not found in dict
```

```
Traceback (most recent call last):
  File "plugin.py", line 292, in actionControlDevice
KeyError: key lowLimit not found in dict
```

Seems there was a small typo on the following two lines of code:
https://github.com/FlyingDiver/Indigo-Masquerade/blob/master/Masquerade.indigoPlugin/Contents/Server%20Plugin/plugin.py#L286

https://github.com/FlyingDiver/Indigo-Masquerade/blob/master/Masquerade.indigoPlugin/Contents/Server%20Plugin/plugin.py#L292

Updating these two lines to say dev.pluginProps["highLimitState"] and dev.pluginProps["lowLimitState"] respectively has fixed my dimmer on/off issues.

**Comment by FlyingDiver** [comment]
Huh.  That code is almost two years old, and no one ever reported that.  Thanks!

**Comment by FlyingDiver** [comment]
Fixed in 0.1.5
